src/index.html

```diff
@@ -6,8 +6,55 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin = "#"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
+
   <body>
-    <h1>Product cards</h1>
+    <article data-qa="card" class="card">
+
+      <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27"
+      >
```

Comment:
- Don't add empty lines between parent and child elements.
- Use 2 spaces for indentation in your file (not a tab character)
![image](https://github.com/mate-academy/layout_product-cards/assets/63243136/b3a81f58-9e05-4000-a19f-047cdc744acb)